src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.model;
+
+public class Car {
+
+    private int position = 0;
+    private String name;
```

Comment:
`
```suggestion
    private final String name;
```
`final`로 해줘도 될것 같아요!

Reply:
그러게요 final 키워들 잘 사용하지 못한 것 같은데 다음 미션 때는 잘 적용하고 싶어요